Mathlib/LinearAlgebra/FiniteDimensional.lean

```diff
@@ -515,20 +516,18 @@ theorem exists_nontrivial_relation_sum_zero_of_rank_succ_lt_card [FiniteDimensio
     -- ∑ (x : V) in t', g x • x = 0.
     simp only [f]
     conv_lhs =>
-      apply_congr
-      skip
+      congr
       rw [ite_smul]
     rw [Finset.sum_ite]
     conv =>
       congr
       congr
-      apply_congr
```

Comment:
This isn't a reasonable substitution; I think we actually want the `apply_congr` tactic ported

Reply:
I opened #3471.

Reply:
I've fixed the use of `apply_congr`. A gotcha is that we need to use `rfl` rather than `skip` inside `conv`, apparently.